lang/nl.js

```diff
@@ -10,9 +10,9 @@ export default {
 	"components.form-element.defaultError": "{label} is ongeldig.",
 	"components.form-element.defaultFieldLabel": "Veld",
 	"components.form-element.input.email.typeMismatch": "E-mailadres is ongeldig",
-	"components.form-element.input.number.rangeError": "Getal moet tussen {min} en {max} liggen.",
-	"components.form-element.input.number.rangeOverflow": "Getal moet lager zijn dan {max}.",
-	"components.form-element.input.number.rangeUnderflow": "Getal moet hoger zijn dan {min}.",
+	"components.form-element.input.number.rangeError": "Getal moet {minExclusive, select, true {greater than} false {greater than or equal to}} {min} en {maxExclusive, select, true {less than} false {less than or equal to}} {max} zijn.",
+	"components.form-element.input.number.rangeOverflow": "Getal moet {maxExclusive, select, true {less than} false {less than or equal to}} {max} zijn.",
+	"components.form-element.input.number.rangeUnderflow": "Getal moet {minExclusive, select, true {greater than} false {greater than or equal to}} {min} zijn.",
```

Comment:
Looks like the stuff inside the `select` didn't get translated.

Reply:
Have we run into this before?  Do we just accept this change and "hope" that it gets translated next time?

Reply:
I don't remember running into this before, but it's one of the first times we've used these more complex syntaxes so probably some of the translators have never seen them before. Worth pinging Jeff S on Slack once it's back up.